benefits/core/migrations/local_fixtures.json

```diff
@@ -115,6 +115,22 @@
       "claims_claim": "calfresh"
     }
   },
+  {
+    "model": "core.enrollmentflow",
+    "pk": 5,
+    "fields": {
+      "system_name": "medicare",
+      "label": "(CST) Medicare Discount",
+      "group_id": "group123",
+      "enrollment_success_template": "enrollment/success--cst.html",
+      "display_order": 1,
+      "claims_provider": 1,
```

Comment:
Since we will need a new `ClaimsProvider` for this pathway, can you please create one now in this PR?

Maybe change the `client_name` of the existing one to something like `benefits-logingov` and then create a new one `benefits-medicaregov`.